scaper/util.py

```diff
@@ -11,6 +11,8 @@
 from .scaper_exceptions import ScaperError
 import scipy
 import numpy as np
+import numbers
+import random
```

Comment:
Why is random imported? Presumably our goal is to remove all use of this, right? I don't think you're using it anywhere in `util.py` anyway... so should be safe/easy to remove this import.

Reply:
Yeah good call. Removed!